### PR TITLE
update dep performance-now to current version && bump version to 3.3.1

### PR DESCRIPTION
**Pull request by soenkekluth**
performance-now had some major changes from 0.2.0 to 2.1.0 that will optimize the performance and stability of raf.
test are passing.
you could think about making this a minor update. i only patched the version to to 3.3.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "raf",
-  "version": "3.3.0",
+  "version": "3.3.1",
```
Comment (ljharb) [inline package.json]: versions shouldn't be bumped in PRs.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "tape": "^4.0.0"
   },
   "dependencies": {
-    "performance-now": "~0.2.0"
+    "performance-now": "~2.1.0"
```
Comment (ljharb) [inline package.json]: this should use `^` instead of `~` for a wider semver range

File: package.json (hunk 1 above)
Comment (soenkekluth) [inline package.json]: you are right - just wanted to save you a minute ;) its reversed.

**Comment by soenkekluth** [comment]
yo what about a magic merge?
one of the things i am using raf if your interested https://www.npmjs.com/package/nextframe

**Comment by ljharb** [comment]
I'm afraid I don't have merge powers; @chrisdickinson?

**Comment by CMTegner** [comment]
Sorry for taking so long, thanks!

**Comment by CMTegner** [comment]
Published in 3.3.2 (accidentally re-published 3.3.0 as 3.3.1, hence 2 instead of 1).

**Pull request merged**